server/model/monitor.js

```diff
@@ -71,23 +71,15 @@ class Monitor extends BeanModel {
 
     /**
      * Return an object that ready to parse to JSON
+     * @param {object} preloadData Include precalculate data in
      * @param {boolean} includeSensitiveData Include sensitive data in
      * JSON
      * @returns {Promise<object>} Object ready to parse
      */
-    async toJSON(includeSensitiveData = true) {
+    async toJSON(preloadData = {}, includeSensitiveData = true) {
```

Comment:
This is now nothing async more in here, right?
=> Please remove the async from this function and make sure that where this is called is not preloaded..

Reply:
@CommanderStorm 
understood. i will remove & test again

Reply:
@CommanderStorm 
i have removed async